controllers/grid/queries/QueryNotesGridHandler.php

```diff
@@ -318,14 +317,12 @@ protected function insertedNoteNotify(Note $note): void
                 continue;
             }
 
-            $mailable = $this->getStageMailable($context, $submission, $title, $note->getContents());
-            $emailTemplate = Repo::emailTemplate()->getByKey($context->getId(), $mailable::getEmailTemplateKey());
-
             $recipient = Repo::user()->get($userId);
-            $mailable->sender($sender)
+            $mailable = $this->getStageMailable($context, $submission)
+                ->sender($sender)
                 ->recipients([$recipient])
-                ->subject(__('common.re').' '.$emailTemplate->getLocalizedData('subject'))
-                ->body($emailTemplate->getLocalizedData('body'));
+                ->subject(__('common.re').' '.$title)
+                ->body($note->getContents());
```

Comment:
Yeah, so, on reflection, I decided to abandon the "simple template", since it doesn't really function in this way. Instead, I changed the default discussion template to the old `notification` template. It's only ever used as a template shown to the user, like other mailables with multiple templates. The body of the message is whatever the user sets it to.

But I can see that this should allow unsubscribe, too, so I added `allowUnsubscribe()` here.